pkgs/development/mobile/cocoapods/update

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -i bash -p bash ruby bundler bundix
+#! nix-shell -i bash -p bash bundix bundler
 
-BUNDIX_CACHE="$(mktemp -d)"
+set -e
 
-rm -f Gemfile.lock
-bundler package --path "$BUNDIX_CACHE"
-bundix --bundle-pack-path="$BUNDIX_CACHE"
-
-rm -rf "$BUNDIX_CACHE"
-rm -rf .bundle
+bundler lock --update
+# For some reson `bundler lock` doesn't take a `--gemfile` flag
```

Comment:
Does the `BUNDLE_GEMFILE` environment variable work?

Reply:
I just tested, looks like the answer is "no". I have to delete Gemfile-beta.lock first as apparently `bundler lock --update` won't backdate a lockfile despite the documentation claiming that flag makes it ignore the current lockfile (🤦🏻‍♀️) but after doing that, passing `env BUNDLE_GEMFILE=Gemfile-beta bundler lock --update --lockfile=Gemfile-beta.lock`  produces a lockfile that's clearly coming from `Gemfile`.

Reply:
Grr, okay. If you open a Bundler bug and add it to the comment, I'm happy to merge this.

Reply:
Huh. I just tested again to make sure and it actually seems like it's working now. I don't know what I did wrong before, since I double-checked my work last time, but if it really does work, I'll update this script to use that.

Reply:
Woo!